[Challenge #1] Média dos alunos

kev-karan opened the pull request.

kev-karan committed:
Finalizando o desafio

Logikoz reviewed (approved):
Boa noite primo! O quê você acha de fazer um plus pra esse desafio? Se você puder ter infinitas notas para fazer essa média, como você faria? Caso também o usuário colocar algum tipo de informação não numérica dentro da sua aplicação, como ela se portaria?

Caso você queira o desafio, tá ai uma boa pedida. Porém caso você não quiser, me sinaliza no post para que eu possa mergear o PR!

kev-karan commented:
> Se você puder ter infinitas notas para fazer essa média, como você faria? Caso também o usuário colocar algum tipo de informação não numérica dentro da sua aplicação, como ela se portaria?

Não tenho certeza de como fazer isso, mas aceito o desafio

kev-karan committed:
correção de input não numerico

impede o usuário de colocar informações não numéricas na aplicação

kev-karan committed:
Finalizando o desafio do desafio

consegui implementar uma forma do usuário poder escolher a quantidade de provas para tirar a media entre elas

kev-karan committed:
Atualização do README.MD
